README.md

```diff
@@ -1,11 +1,53 @@
 # Tektoncd Operator
 
-Kubernetes [Operator](https://operatorhub.io/getting-started) to
-manage installations, updates and removal of Tektoncd projects
-(pipeline, dashboard, …)
-
-> The goal of an Operator is to put operational knowledge into
-> software. Previously this knowledge only resided in the minds of
-> administrators, various combinations of shell scripts or automation
-> software like Ansible. It was outside of your Kubernetes cluster and
-> hard to integrate. With Operators, we changed that.
+Kubernetes [Operator](https://operatorhub.io/getting-started) to manage installations, updates and removal of Tektoncd projects (pipeline, dashboard, …)
+
+The following steps will install [Tekton Pipeline](https://github.com/tektoncd/pipeline) and configure it appropriately for your cluster.
+1. Create namespace: `tekton-operator`  
+
+    `kubectl create namespace tekton-operator`
+    
+2. Apply Operator crd  
+
+    `kubectl apply -f deploy/crds/*_crd.yaml`
+    
+3. Deploy the Operator  
+
+    `kubectl -n tekton-operator apply -f deploy/`  
+    
```

Comment:
I remember we have a demo saying we can either go with the auto-install or manual install of CR?
Shall we add the instruction on it?

Reply:
Done